doc/visual-programming/source/widgets/data/mergedata.md

```diff
@@ -12,24 +12,79 @@ Merges two datasets, based on values of selected attributes.
 
 - Data: dataset with features added from extra data
 
-The **Merge Data** widget is used to horizontally merge two datasets, based on values of selected attributes. In the input, two datasets are required, data and extra data. The widget allows selection of an attribute from each domain, which will be used to perform the merging. The widget produces one output. It corresponds to instances from the input data to which attributes from input extra data are appended.
+The **Merge Data** widget is used to horizontally merge two datasets, based on the values of selected attributes (columns). In the input, two datasets are required, data and extra data. The widget allows selection of one or more attributes from each domain, which will be used to perform the merging. The widget produces one output. It corresponds to the instances from the input data to which attributes (columns) from input extra data are appended.
 
-Merging is done by values of selected (merging) attributes. First, the value of the merging attribute from Data is taken and instances from Extra Data are searched for matching values. If more than a single instance from Extra Data was to be found, the attribute is removed from available merging attributes.
+Merging is done by values of selected (merging) attributes. First, the value of the merging attribute from Data is taken and instances from Extra Data are searched for matching values. If the selected attribute does not contain unique values (in other words, the attribute has duplicate values), the attribute is removed from the available merging attributes.
```

Comment:
This is no longer true because now it suffices that a combination of attributes yields unique values. Now, it's an error if the chosen combination of values is not unique.

I'd also say "matching" instead of merging.

Actually, I'd remove this paragraph and change the previous one.